tools/goctl/model/sql/converter/types.go

```diff
@@ -305,7 +305,7 @@ func mayConvertNullType(goDataType string, isDefaultNull, unsigned, strict bool)
 	case "time.Time":
 		return "sql.NullTime"
 	default:
-		if unsigned {
+		if unsigned && strict {
```

Comment:
Here you should consider whether to use goctl config. If so, don't use strict control, otherwise it will be regarded as an old function and can be repaired compatible.

Reply:
convertDatatypeWithConfig is already called in ConvertStringDataType before mayConvertNullType, there's no need to modify mayConvertNullType to check goctl config again.